exploration/expression-attributes.md

```diff
@@ -88,10 +60,10 @@ At least the following expression attributes should be considered:
     whether the expression should or should not be localised.
     The values here correspond to those used for this property in HTML and elsewhere.
 
-    > Example embedding a non-translatable French literal in an English message:
+    > Example embedding a non-translatable CLI command in a message:
     >
     > ```
-    > In French, "{|bonjour| @locale=fr @translate=no}" is a greeting
+    > Use {+code @translate=no}git ls-files{-code} to list all files in a repository.
```

Comment:
That is probably incorrect.

`@translate` is not a runtime attribute, so it is intended for translators (humans or machine, it does not matter).

Until now the placeholders and markup don't have side-effects on anything outside them.
A translator can move around placeholders / markup, maybe delete it. But it is free to changer any text outside them.
But this proposal changes that.

We already have a way to achieve this:
```
Use {+code}{|git ls-files|}{-code} to list all files in a repository.
```